src/classes/CMT_MetadataAPI.cls

```diff
@@ -99,6 +99,15 @@ public class CMT_MetadataAPI {
      * @param mtToDeploy List of CustomMetadata objects to deploy
      */
     public static void deployMetadata(String jobId, List<Metadata.CustomMetadata> mtToDeploy) {
```

Comment:
@ptewson-sfdo - Do you think overloading the deployMetadata method to add a flag for the result handler to use later is a good idea? HandleResult method needs a way to know which custom settings to update

Reply:
Ah I see - I think there may be a better way to handle this but I will have to chew on it. At very least I think there may be a better name for that variable than "isHelpTextReleated", but there's some options ... if you can hunt down the author of this comment:

// THIS WILL NEED TO BE REFACTORED IF THIS CMT DEPLOYMENT CODE IS USED BY SOMETHING OTHER THAN ROLLUPS

... maybe they would have ideas? One option would actually be to define a class hierarchy for "Metadataday callback support" or whatever but that feels like massive overkill, so maybe just passing in a variable to indicate what we are doing there like you are already is best. 

Would that make sense as an enumeration instead of a boolean just in case we support additional types in the future? And maybe the contents of the separate branches could move into their own methods just so it's a bit more obvious about how that's getting handed off?